From horms@verge.net.au  Tue Dec 22 11:04:03 2009
From: Simon Horman <horms@verge.net.au>
Date: Mon, 21 Dec 2009 15:07:10 +1100
Subject: Staging: rtl8192su: Build should select WIRELESS_EXT
To: devel@driverdev.osuosl.org, linux-kernel@vger.kernel.org
Cc: Greg Kroah-Hartman <gregkh@suse.de>, Randy Dunlap <randy.dunlap@oracle.com>, Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Message-ID: <20091221040710.GA22636@verge.net.au>
Content-Disposition: inline


The prevailing idiom is to select WIRELESS_EXT not depend on it.

Depending leaves this driver in a situation where it can only
be built if another driver that selects WIRELESS_EXT has been enabled.

Signed-off-by: Simon Horman <horms@verge.net.au>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rtl8192su/Kconfig |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/staging/rtl8192su/Kconfig
+++ b/drivers/staging/rtl8192su/Kconfig
@@ -1,6 +1,7 @@
 config RTL8192SU
 	tristate "RealTek RTL8192SU Wireless LAN NIC driver"
 	depends on PCI && WLAN && USB
-	depends on WIRELESS_EXT
+	select WIRELESS_EXT
+	select WEXT_PRIV
 	default N
 	---help---
